### ISSUE TITLE
Black background in transparent block faces in Vibrant Shaders (Does not occour in Enhanced Default)

### ISSUE THREAD

**Issue by ryu-ketsueki** [Sildurs-shaders/sildurs-shaders.github.io#193]
There are some mods that uses cube faces with transparent faces or partially transparent, with some pixels in and some out. One of these mods is Figura, which reads the Blockbench project files and render them as the player model. However, if Vibrant Shaders is running, some black overlay appear on the face facing foward. Here are some pictures of it happening:

![image](https://user-images.githubusercontent.com/39316730/145879351-e0d95626-86ec-4d24-8968-d37d34c0df98.png)
![image](https://user-images.githubusercontent.com/39316730/145880932-8f076e81-0e73-4987-a28e-01ad9c71f1f4.png)

Curiously, if the face is facing the opposite side, that effect does not happen:

![image](https://user-images.githubusercontent.com/39316730/145879490-bb719138-85f8-4ef5-a741-6a8aceb9f8ca.png)

Same for Enhanced Default:
![image](https://user-images.githubusercontent.com/39316730/145881058-f7e33217-34d3-4f33-9b7f-15d987a1b153.png)
![image](https://user-images.githubusercontent.com/39316730/145881115-8bb8c5f3-acc6-4403-8242-c632e70cdc51.png)

I think something could be done to check if a cube face have something transparent on them, kinda like the leaves of a tree.

By the way, the screenshots were taken using Vibrant Lite and Enhanced Default Fancy

**Comment by Sildurs-shaders** [comment]
Shadersmod can't check for that, so it's an issue with transparency and the shadersmod picking things up wrongly.

**Comment by ryu-ketsueki** [comment]
> Shadersmod can't check for that, so it's an issue with transparency and the shadersmod picking things up wrongly.

I was chatting with some people at the mod's server and asked what could be causing this and, while I found a solution for my problem, I learned that some shaderpacks may use a different way of applying some shaders, with one of them even theorizing it was multiply.

As for the solution of my problem, I noticed it was actually related to the fullbright texture, also called emission, which had a black background that the renderer removed. While that does not explain why one shaderpack doesn't change that behavior of removing the black background with the other actually bringing it back, the solution was just as easy as removing the black background entirely and the fullbright feature still works without it. I'm only posting this in case someone stumbles on the same problem.
